File: XOOP/Seminar005/Homework/User/PhoneUser.java
package User;

public class PhoneUser {
    protected String firstName;
    protected String lastName;
    protected String patronymic;
    protected int phoneNumber;
    protected int dayOfBirth;
    protected int monthOfBirth;
    protected int yearOfBirth;

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getPatronymic() {
        return patronymic;
    }

    public String getFullName() {
        return String.format("%s %s %s", lastName, firstName, patronymic);
    }

    public int getPhoneNumber() {
        return phoneNumber;
    }

    public int getDayOfBirth() {
        return dayOfBirth;
    }

    public int getMonthOfBirth() {
        return monthOfBirth;
    }

    public int getYearOfBirth() {
        return yearOfBirth;
    }

    public String getFullInformation() {
        return String.format("%s: %s %s %s: %s/%s/%s", phoneNumber, lastName, firstName, patronymic, dayOfBirth,
                monthOfBirth,
                yearOfBirth);
    }

    public void setUserFirstName(String firstName) {
        this.firstName = firstName;
    }

    public void setUserLastName(String lastName) {
        this.lastName = lastName;
    }

    public void setUserPatronymic(String patronymic) {
        this.patronymic = patronymic;
    }

    public void setUserPhoneNumber(int phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public void setUserDayOfBirth(int dayOfBirth) {
        this.dayOfBirth = dayOfBirth;
    }

    public void setUserMonthOfBirth(int monthOfBirth) {
        this.monthOfBirth = monthOfBirth;
    }

    public void setUserYearOfBirth(int yearOfBirth) {
        this.yearOfBirth = yearOfBirth;
    }

}

File: XOOP/Seminar007/Homework/User/PhoneUser.cs
public class PhoneUser
{
    protected string firstName = "";
    protected string lastName = "";
    protected string patronymic = "";
    protected int phoneNumber;
    protected int dayOfBirth;
    protected int monthOfBirth;
    protected int yearOfBirth;

    public string getFirstName()
    {
        return firstName;
    }

    public string getLastName()
    {
        return lastName;
    }

    public string getPatronymic()
    {
        return patronymic;
    }

    public string getFullName()
    {
        return string.Format("{0} {1} {2}", lastName, firstName, patronymic);
    }

    public int getPhoneNumber()
    {
        return phoneNumber;
    }

    public int getDayOfBirth()
    {
        return dayOfBirth;
    }

    public int getMonthOfBirth()
    {
        return monthOfBirth;
    }

    public int getYearOfBirth()
    {
        return yearOfBirth;
    }

    public string getFullInformation()
    {
        return string.Format("{0}: {1} {2} {3}: {4}/{5}/{6}", phoneNumber, lastName, firstName, patronymic, dayOfBirth,
                monthOfBirth,
                yearOfBirth);
    }

    public void setUserFirstName(string firstName)
    {
        this.firstName = firstName;
    }

    public void setUserLastName(string lastName)
    {
        this.lastName = lastName;
    }

    public void setUserPatronymic(string patronymic)
    {
        this.patronymic = patronymic;
    }

    public void setUserPhoneNumber(int phoneNumber)
    {
        this.phoneNumber = phoneNumber;
    }

    public void setUserDayOfBirth(int dayOfBirth)
    {
        this.dayOfBirth = dayOfBirth;
    }

    public void setUserMonthOfBirth(int monthOfBirth)
    {
        this.monthOfBirth = monthOfBirth;
    }

    public void setUserYearOfBirth(int yearOfBirth)
    {
        this.yearOfBirth = yearOfBirth;
    }

}
